Makefile

```diff
@@ -117,6 +117,24 @@ image-push: ## Push container image
 .PHONY: image
 image: image-build image-push ## Build and push docker image with the manager.
 
+.PHONY: update-manifests
+update-manifests: ## Update image tags in manifests
+	sed -i 's/newTag: .*/newTag: $(VERSION)/' config/manager/kustomization.yaml
+	@echo "After update:"
+	@sed -n '/newTag:/p' config/manager/kustomization.yaml
+
+.PHONY: release
+release: ## Create a new release
+	@read -p "Enter new version (e.g., vX.Y.Z): " VERSION && \
+	$(MAKE) update-manifests VERSION=$$VERSION && \
+	git add config/ && \
+	git commit -m "release: $$VERSION" && \
+	git tag -a "$$VERSION" -m "Release $$VERSION" && \
+	$(MAKE) update-manifests VERSION="latest" && \
+	git add config/ && \
+	git commit -m "release: next iteration"
+	@echo "Done, don't forget to push. When pushing, add --tags to your call."
```

Comment:
Good stuff! I have just one idea we can consider. `VERSION` variable is already defined. So we can leverage that.

It's either a value of the tag if that's `HEAD` or `tag-next` if we have more commits on top of latest tag.

so we could actually have something like that:

```sh
❯ make release -e VERSION=v0.0.6

Releasing version v0.0.6

[main ced14b5] release: v0.0.6
 1 file changed, 1 insertion(+), 1 deletion(-)
[main f566f1f] release: next iteration
 1 file changed, 1 insertion(+), 1 deletion(-)
Done, don't forget to push. When pushing, add --tags to your call.
❯ make release

Releasing version v0.0.6-next

[main dd70831] release: v0.0.6-next
 1 file changed, 1 insertion(+), 1 deletion(-)
[main 13496f7] release: next iteration
 1 file changed, 1 insertion(+), 1 deletion(-)
Done, don't forget to push. When pushing, add --tags to your call.
❯ git lo
13496f7  (HEAD -> main) release: next iteration  (by bartoszmajsak)
dd70831  (tag: v0.0.6-next) release: v0.0.6-next  (by bartoszmajsak)
f566f1f  release: next iteration  (by bartoszmajsak)
ced14b5  (tag: v0.0.6) release: v0.0.6  (by bartoszmajsak)
73feac6  release: next iteration  (by bartoszmajsak)
```

`make release -e VERSION=v0.0.6` is automation friendly. WDYT?

Here's what I came up with:

```suggestion
##@ Release
define update-manifests
	sed -i 's/newTag: .*/newTag: $(1)/' config/manager/kustomization.yaml
endef

.PHONY: release
release: ## Create a new release
	$(call header,"Releasing version $(VERSION)")
	@$(call update-manifests,$(VERSION)); \
	git add config/ && \
	git commit -m "release: $(VERSION)" && \
	git tag -a "$(VERSION)" -m "Release $(VERSION)"; \
	$(call update-manifests,latest); \
	git add config/ && \
	git commit -m "release: next iteration"
	@echo "Done, don't forget to push. When pushing, add --tags to your call."

```